Detectors/GlobalTracking/src/MatchTPCITS.cxx

```diff
@@ -1825,14 +1822,13 @@ int MatchTPCITS::followABSeed(const o2::track::TrackParCov& seed, const ITSChipC
     // coordinates xCross,yCross,zCross for this central chipIDguess, although we are going to check also neighbours
     float t = 1e9, xCross, yCross;
     const auto& chipC = lad.chips[chipIDguess];
-    bool res = mFieldON ? chipC.xyEdges.circleCrossParam(trcCircle, t) : chipC.xyEdges.lineCrossParam(trcLinPar, t);
     chipC.xyEdges.eval(t, xCross, yCross);
     float dx = xCross - xCurr, dy = yCross - yCurr, dst2 = dx * dx + dy * dy, dst = sqrtf(dst2);
     // Z-step sign depends on radius decreasing or increasing during the propagation
     float zCross = seedC.getZ() + seedC.getTgl() * (dst2 < 2 * (dx * xCurr + dy * yCurr) ? dst : -dst);
 
-    for (int ich = -1; ich < 2; ich++) {
-      int chipID = chipIDguess + ich;
+    for (size_t ich = -1; ich < 2; ich++) {
```

Comment:
This one is probably wrong: `size_t ich = -1`

Reply:
Sure, my bad.